examples/hooks/App.css

```
@@ -16,6 +16,36 @@ body {
   margin: 0 auto;
 }
 
+.Container {
+  display: grid;
+  align-items: flex-start;
+  grid-template-columns: 200px 1fr;
+  gap: 0.5rem;
+}
+
+.Search {
+  display: grid;
+  gap: .5rem;
+}
+
+.Search-header {
+  display: flex;
+}
+
+.Search-box {
```

To follow the component name:

```suggestion
.SearchBox {
```